### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by michaelortmann**
Found by: michaelortmann
Patch by: michaelortmann
Fixes: 

One-line summary:
remove dead code

Additional description (if needed):
See also commit 66f6d8734c88d456573665d67e5260f72bc5619c

Test cases demonstrating functionality (if applicable):
.who output of channels on multiple lines unchanged:
**Before:**
```
.who BotB
[02:03:35] tcl: builtin dcc call: *dcc:who -HQ 1 BotB
[02:03:35] #-HQ# who BotB
*** [BotB] Channels: #00000000000000000000, #00000000000000000001, #00000000000000000002, #00000000000000000003, #00000000000000000004, #00000000000000000005, #00000000000000000006, #00000000000000000007, #00000000000000000008, #00000000000000000009, #00000000000000000010, #00000000000000000011, #00000000000000000012, #00000000000000000013, #00000000000000000014, #00000000000000000015, #00000000000000000016, #00000000000000000017,
*** [BotB]           #bOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOO
*** [BotB] Version: eggdrop v1.9.0+serverlen
*** [BotB] Admin: Lamer <email: lamer@lamest.lame.org>
*** [BotB] Party line members: (* = owner, + = master, % = botmaster, @ = op, ^ = halfop)
*** [BotB]   *-HQ             llama@console
*** [BotB] Bots connected:
*** [BotB]   <-+BotA            eggdrop v1.9.0+serverlen <I.didn't.edit.my.config.file.net>
```
**After:**
```
.who BotB
[02:15:27] tcl: builtin dcc call: *dcc:who -HQ 1 BotB
[02:15:27] #-HQ# who BotB
*** [BotB] Channels: #00000000000000000000, #00000000000000000001, #00000000000000000002, #00000000000000000003, #00000000000000000004, #00000000000000000005, #00000000000000000006, #00000000000000000007, #00000000000000000008, #00000000000000000009, #00000000000000000010, #00000000000000000011, #00000000000000000012, #00000000000000000013, #00000000000000000014, #00000000000000000015, #00000000000000000016, #00000000000000000017,
*** [BotB]           #bOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOOmbOO
*** [BotB] Version: eggdrop v1.9.0+serverlen
*** [BotB] Admin: Lamer <email: lamer@lamest.lame.org>
*** [BotB] Party line members: (* = owner, + = master, % = botmaster, @ = op, ^ = halfop)
*** [BotB]   *-HQ             llama@console
*** [BotB] Bots connected:
*** [BotB]   <-+BotA            eggdrop v1.9.0+serverlen <I.didn't.edit.my.config.file.net>
```

### CODE REVIEW

**Comment by vanosg** [comment]
Please give a description of what is broken and what you are trying to fix, along with applicable demonstrated test cases showing what didn't work, what now 'works', and the remaining functionality still working. Thanks!

**Comment by michaelortmann** [comment]
The code touched by this PR was broken for gcc 10 beta, which  reported a compiler warning. That warning  is gone with gcc 10 final. But: when i looked at the code for remote_tell_who(), see https://github.com/eggheads/eggdrop/pull/877/files it looks like the reported channels as a string could be truncated and/or lines could be ended by "..." with code line `strcpy(s + ssize - trunc, " ...");`. But this code looks dead, or can you reproduce such "..."? So i ripped off this dead code with this PR. There is no user visible change, see test case in this PR header.

Maybe its a different story, and that code shouldnt be dead. Can you give me an example, where the "..." string comes into play? I hope, you can remember, because it was only recently added with commit 66f6d8734c88d456573665d67e5260f72bc5619c. Maybe the code is only broken and needs some more test/fixing. Pls help me figuring this out.

**Comment by vanosg** [comment]
So there's three things in play here- this code is truncating a line if it is longer than 450 characters. But preceding it, there is a check that sees how many channels can be crammed onto one line, and if it is longer than 450, it moves the next channel to the next line. Finally, CHANNELLEN is 80.

Now to the particular check you're referencing, it is a last check that is intended to truncate a single line if the channel is over 440 characters. Now with CHANNELLEN being 80 as it is now, this won't ever get hit, but as we start to muck with these other weird things (like, twitch for example) there could be an opportunity where a channel could hit this limit. So, no matter how unlikely that is, I think we can look at this code block more as 'future proofing' the code base against something we don't yet know. I'll add a comment to the code base reflecting this.

So in short- yeah, it probably won't ever get hit, but a future mod elsewhere in code could, so I think we're ok leaving it. I'll close this for now, but let me know if I'm off base with what you're seeing. Good catch, though!
